src/components/common/applicant-info-3.vue

```diff
@@ -68,6 +68,15 @@
                         hide-details="auto"
                         label="Last Name" />
         </v-col>
+        <v-col cols="2" v-if="!getEditMode"></v-col>
```

Comment:
This should be in its own v-row (as there are 12 columns in a row, and here you are overflowing the first row in order to get another row... which is not the right way to do it).

Reply:
ok. You're right. Changing...